packages/create/src/recipes/remix/index.ts

```diff
@@ -0,0 +1,56 @@
+import path from "node:path";
+import debug from "debug";
+import { updatePackage } from "write-package";
+
+import type { BaseOptions, Recipe } from "../types.js";
+import { copyTemplateFiles } from "../../utils.js";
+
+const logger = debug("@edgedb/create:recipe:remix");
+
+const recipe: Recipe = {
+  skip(opts: BaseOptions) {
+    return opts.framework !== "remix";
+  },
+  async apply({ projectDir, useEdgeDBAuth }: BaseOptions) {
+    logger("Running remix recipe");
+
+    const dirname = path.dirname(new URL(import.meta.url).pathname);
+    await copyTemplateFiles(path.resolve(dirname, "./template"), projectDir);
+
+    await updatePackage(projectDir, {
+      sideEffects: false,
+      type: "module",
+      scripts: {
+        dev: "remix dev --manual",
+        build: "remix build",
+        lint: "eslint --ignore-path .gitignore --cache --cache-location ./node_modules/.cache/eslint .",
+        start: "remix-serve ./build/index.js",
+        typecheck: "tsc",
+      },
+      dependencies: {
+        ...(useEdgeDBAuth ? { "@edgedb/auth-nextjs": "^0.1.0-beta.1" } : {}),
```

Comment:
```suggestion
        ...(useEdgeDBAuth ? { "@edgedb/auth-remix": "^0.1.0-beta.1" } : {}),
```

Also, I wonder if there is a version specifier that will install whatever version is actually published 🤔

Reply:
oh this already got merged ( I enabled auto merge), will create another PR now